bin/ghe-restore

```diff
@@ -9,21 +9,22 @@
 #/ <https://enterprise.github.com/help/articles/adding-an-ssh-key-for-shell-access>
 #/
 #/ OPTIONS:
-#/   -f | --force      Don't prompt for confirmation before restoring.
-#/   -c | --config     Restore appliance settings and license in addition to
-#/                     datastores. Settings are not restored by default to
-#/                     prevent overwriting different configuration on the
-#/                     restore host.
-#/   -v | --verbose    Enable verbose output.
-#/   -h | --help       Show this message.
-#/        --version    Display version information and exit.
-#/   -s <snapshot-id>  Restore from the snapshot with the given id. Available
-#/                     snapshots may be listed under the data directory.
-#/   <host>            The <host> is the hostname or IP of the GitHub Enterprise
-#/                     instance. The <host> may be omitted when the
-#/                     GHE_RESTORE_HOST config variable is set in backup.config.
-#/                     When a <host> argument is provided, it always overrides
-#/                     the configured restore host.
+#/   -f | --force         Don't prompt for confirmation before restoring.
+#/   -c | --config        Restore appliance settings and license in addition to
+#/                        datastores. Settings are not restored by default to
+#/                        prevent overwriting different configuration on the
+#/                        restore host.
+#/        --skip-pages    Skip restore of GitHub Pages artifacts.
```

Comment:
I think @itillis was hoping this could be figured out dynamically at restore time instead of requiring users to pass the flag in again. If pages aren't present in the backup (intentionally or not), we can trigger a rebuild -- are we trying to avoid that unnecessary rebuild command in the case where there actually aren't any to rebuild?

Reply:
Yeah, so if there were no pages in backup (and if you choose to skip) we will add a rebuild step for pages (after that script is done), which will come in a separate PR.

Reply:
Agree. Or a `sentinel` file if necessary

Reply:
I don't think a sentinel file is needed in this case. We've moved to using config setting instead.